src/lib/nfv/selftest.sh

```diff
@@ -70,6 +70,9 @@ function stop_bench_env {
 
     # Give VMs and snabbnfv-traffic time to shut down.
     sleep 5
+
+    # Clean up temporary config location.
+    rm -f "$TESTCONFPATH"
```

Comment:
Yes probably, I only learned about the existence of traps from `bench_env`, excuse my ignorance. I will open a PR to fix this.